src/dos/dos_mscdex.cpp

```diff
@@ -308,7 +308,8 @@ int CMscdex::AddDrive(Bit16u _drive, char* physicalPath, Bit8u& subUnit)
 		break;
 	case MountType::DIRECTORY:
 		LOG(LOG_MISC,LOG_NORMAL)("MSCDEX: Mounting directory as cdrom: %s", physicalPath);
-		LOG(LOG_MISC,LOG_NORMAL)("MSCDEX: You wont have full MSCDEX support!");
+		LOG(LOG_MISC, LOG_NORMAL)
+		("MSCDEX: You won't have full MSCDEX support!");
```

Comment:
@okias -- can you revert (or unwrap) these `LOG(..)(..)` statements? 
 - dos_mscdec.cpp: 311
 - drive_fat.cpp: 587
 - keyboard.cpp: 129

They are so far the only thing that CLang format doesn't handle properly.  (Sorry for not mentioning this!)

Reply:
yeah, it looked weird, I was wondering, why you guys insist on these clang changes :D

Reply:
If we could exclude just that specific `LOG()()` pattern, we would :-) 
Otherwise, we're trying to steadily migrate the code base to a consistent and mechanically-applied format.